governator-test/src/main/java/com/netflix/governator/guice/test/AnnotationBasedTestInjectorManager.java

```diff
@@ -94,6 +96,20 @@ public InjectorCreationMode getInjectorCreationMode() {
     protected LifecycleInjector createInjector(List<Module> modules, List<Module> overrides) {
         return InjectorBuilder.fromModules(modules).overrideWith(overrides).createInjector();
     }
+    
+    protected MockHandler createMockHandlerForTestClass(Class<?> testClass, Class<? extends MockHandler> defaultMockHandler) {
```

Comment:
I should check the parent classes and override with any values given on child classes so that its consistent with all of the other annotation behavior. Good catch.